main.go

```
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/erikjohnston/gotest/media"
+)
+
+func main() {
+	fmt.Println("Starting!")
+
+	downloadServer := new(media.DownloadServer)
+	downloadServer.RepositoryStorePrefix = "downloads"
+	downloadServer.MaxSize = 61440
+	downloadServer.LocalServerName = "localhost"
+
+	uploadServer := new(media.UploadServer)
+	uploadServer.RepositoryStorePrefix = "downloads"
+	uploadServer.MaxSize = 61440
+	uploadServer.LocalServerName = "localhost"
+
+	serverMux := http.NewServeMux()
+	serverMux.Handle("/download/", http.StripPrefix("/download", downloadServer))
+	serverMux.Handle("/upload", uploadServer)
+
+	server := new(http.Server)
+	server.Addr = ":12345"
+	server.Handler = serverMux
+
+	server.ListenAndServe()
```

You want to:

log.Fatal(server.ListenAndServer())

so that if it fails, you get a bad status code and an error message, rather than silently ignoring the error returned from ListenAndServe.

The function doesn't return unless there's an error, so just log.Fatal-ing